packages/shortcuts-extension/schema/plugin.json

```diff
@@ -157,6 +157,15 @@
       },
       "type": "object"
     },
+    "filemenu:persist-and-save": {
+      "default": { },
+      "properties": {
+        "command": { "default": "filemenu:persist-and-save" },
+        "keys": { "default": ["Accel Shift S"] },
```

Comment:
This now conflicts with the "Save As..." shortcut (sorry, I know your PR predates that!). There is a lot of precedence for "Save As..." having `Accel Shift S` as a shortcut. Can we choose something else?

Reply:
I changed to `Accel Ctrl S` which works for me on mac/chrome.